src/node_config.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SRC_NODE_CONFIG_H_
+#define SRC_NODE_CONFIG_H_
+
+#include "node.h"
+
+namespace node {
+
+#if defined(NODE_HAVE_I18N_SUPPORT)
+  extern bool flag_icu_data_dir;
+#endif
+
+namespace config {
+
+}  // namespace config
```

Comment:
Yeah, I was considering that also. I added this initially because I figured there'd be more in it.